scripts/max.davinci.setup.sh

```diff
@@ -29,9 +29,9 @@ export BASE=${HOME}/degas/habanero-upc
 ################################################
 
 # COMPILERS (fixed values unless you are building on Edison)
-export CXX=mpic++
-export CXX_LINKER=mpic++
-export CC=mpicc
+export CXX=mpiicpc
```

Comment:
Yeah, that's why this is explicitly a "max.davinci" setup script, i.e. I didn't replace setup.sh with this. I checked this in the case that others getting set up on DAVINCI might find it useful.